ydb/src/grpc_wrapper/raw_topic_service/common/metering_mode.rs

```diff
@@ -0,0 +1,17 @@
+#[derive(serde::Serialize)]
+pub(crate) enum RawMeteringMode {
+    UseDefault,
```

Comment:
It wil be better to use protospec defined names (with truncated prefixes), because it will be same words for same values in all places.

In this case it better to use Unspecified instead of UseDefault.